src/init.c

```diff
@@ -29,6 +35,10 @@ void R_init_cleancall(DllInfo *dll) {
 
   fns_dot_call = Rf_findVar(Rf_install(".Call"), R_BaseEnv);
 
-  R_RegisterCCallable("cleancall", "r_with_cleanup_context", (DL_FUNC) &r_with_cleanup_context);
-  R_RegisterCCallable("cleancall", "r_push_cleanup", (DL_FUNC) &r_push_cleanup);
+  R_RegisterCCallable("cleancall", "r_with_cleanup_context",
+		      (DL_FUNC) &r_with_cleanup_context);
+  R_RegisterCCallable("cleancall", "r_call_on_exit",
+		      (DL_FUNC) &r_call_on_exit);
+  R_RegisterCCallable("cleancall", "r_call_on_nonlocal_exit",
+		      (DL_FUNC) &r_call_on_nonlocal_exit);
```

Comment:
Indentation looks off. Ah it seems you're using tabs. Can you untabify the whole PR please?